app/assets/javascripts/quby/answers/validation.js

```diff
@@ -126,7 +126,7 @@
                   if(value === undefined || value == ""){
                       continue;
                   }
-                  var rgx = /(\s*-?[1-9]+[0-9]*\s*|\s*-?[0-9]?\s*)/;
+                  var rgx = /(\s*-?[1-9]+[0-9]+\s*|\s*-?[0-9]\s*)/;
```

Comment:
Zelfs al is het niet waarschijnlijker, het risico van een typo weegt zwaarder dan het gebruiksgemak van leidende nullen toestaan.